common/build-helper/rust.sh

```diff
@@ -51,3 +51,6 @@ export OPENSSL_NO_VENDOR=1
 
 # pcre2-sys, only necessary for musl targets
 export PCRE2_SYS_STATIC=0
+
+# zstd-sys
+export ZSTD_SYS_USE_PKG_CONFIG=1
```

Comment:
I'm using this over https://github.com/void-linux/void-packages/pull/43188/files#diff-7ab3afd3c6889e666cf13de92ce9f6a1aa3ffa61178fdf079813e37dd94b5165L60-L63, because it means that we get it automatically everywhere and the nushell patches are a PITA to maintain